react/SelectBox/SelectBox.jsx

```diff
@@ -200,6 +200,7 @@ const ActionsOption = ({ actions, ...props }) => (
     <span className={withPrefix(props.cx, styles['select-option__actions'])}>
       {actions.map((action, index) => (
         <Icon
+          {...action.iconProps}
```

Comment:
can you update the proptype https://github.com/cozy/cozy-ui/blob/06a7a5de700af37800cb55df53308fa60a5bb062/react/SelectBox/SelectBox.jsx#L219 ?